### PR TITLE
Infer lookback in the default EMP config from the withdrawal liveness of the EMP

### PR DESCRIPTION
**Pull request by mrice32**

### CODE REVIEW

File: financial-templates-lib/price-feed/CreatePriceFeed.js [hunk 1]
```diff
@@ -186,12 +186,17 @@ async function createUniswapPriceFeedForEmp(logger, web3, networker, getTime, em
...
+  const lookback = Number((await emp.methods.liquidationLiveness().call()).toString());
+
   // TODO: maybe move this default config to a better location.
   const defaultConfigs = {
     "ETH/BTC": {
       type: "medianizer",
       pair: "ethbtc",
-      lookback: 7200,
+      lookback,
       minTimeBetweenUpdates: 60,
```
Comment (nicholaspai) [inline financial-templates-lib/price-feed/CreatePriceFeed.js:200]: I see you passed `minTimeBetweenUpdates` in the `config` object param, why not read it dynamically from that?

Comment (chrismaree) [reply financial-templates-lib/price-feed/CreatePriceFeed.js:200]: perhaps because this is a default?

Comment (mrice32) [reply financial-templates-lib/price-feed/CreatePriceFeed.js:200]: Yeah, exactly. These are the default values. In the test, I override the minTimeBetweenUpdates to test the override functionality.

Comment (mrice32) [reply financial-templates-lib/price-feed/CreatePriceFeed.js:200]: @nicholaspai lmk if this looks good to you.

Comment (nicholaspai) [reply financial-templates-lib/price-feed/CreatePriceFeed.js:200]: looks good

**Pull request merged**